File: QiitaReader/app/src/main/java/jp/blackawa/example/qiitareader/QiitaService.java
package jp.blackawa.example.qiitareader;

import java.util.List;

import jp.blackawa.example.qiitareader.entity.PostEntity;
import retrofit2.http.GET;
import retrofit2.http.Query;
import rx.Observable;

public interface QiitaService {
    @GET("items")
    Observable<List<PostEntity>> listItems(
            @Query("page") int page,
            @Query("per_page") int perPage
    );
}

File: QiitaReader/app/src/main/java/jp/blackawa/example/qiitareader/QiitaService.kt
package jp.blackawa.example.qiitareader

import jp.blackawa.example.qiitareader.entity.PostEntity
import retrofit2.http.GET
import retrofit2.http.Query
import rx.Observable

interface QiitaService {
    @GET("items")
    fun listItems(
            @Query("page") page: Int,
            @Query("per_page") perPage: Int
    ): Observable<List<PostEntity>>
}
